lib/tapioca/compilers/dsl/active_resource.rb

```diff
@@ -0,0 +1,127 @@
+# typed: true
+# frozen_string_literal: true
+
+require "parlour"
+
+begin
+  require "active_resource"
+rescue LoadError
+  return
+end
+
+module Tapioca
+  module Compilers
+    module Dsl
+      # `Tapioca::Compilers::Dsl::ActiveResource` decorates RBI files for subclasses of
+      # `ActiveResource::Base` which declare `schema` fields
+      # (see https://api.rubyonrails.org/v3.2.6/classes/ActiveResource/Base.html).
```

Comment:
```suggestion
      # (see https://github.com/rails/activeresource).
```
would be a better link.